pylmod/gradebook.py

```diff
@@ -82,6 +82,64 @@ def get_gradebook_id(self, gbuuid):
             raise PyLmodUnexpectedData(failure_messsage)
         return gradebook['data']['gradebookId']
 
+    def get_options(self, gradebook_id):
```

Comment:
This seems like a a different logical change than what this PR is doing as there are no dependencies either way from this to get_staff or get_staff to this.  Open this as another PR?